src/components/uploader/js/Uploader.js

```diff
@@ -206,7 +207,13 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
         var numFilesInBatch = batch.files.length;
         var fileLabelStr = fluid.uploader.fileOrFiles(that, numFilesInBatch);
 
-        var uploadingSize = batch.totalBytesUploaded + that.queue.sizeOfUploadedFiles();
+        var uploadingSize = that.queue.sizeOfUploadedFiles();
+        if (previoussizeOfUploadedFiles !== uploadingSize){
+                        batch.totalBytesUploaded = 0;                             
+        //When a file is completely uploaded, then its size is added to uploadingSize from both that.queue.sizeOfUploadedFiles() and batch.totalBytesUploaded 
```

Comment:
Can you improve this comment? also, place it before the code block where you refer to.